tests/abi/testabi_rel.c

```diff
@@ -95,6 +95,11 @@ static void test_keyCmp (void)
 	succeed_if (keyCmp (k1, k2) < 0, "find_me is smaller");
 	succeed_if (keyCmp (k2, k1) > 0, "find_me is smaller");
 
+	keySetName (k1, "user:user_a/a");
+	keySetName (k2, "user:user_b/a");
+	succeed_if (keyCmp (k1, k2) == 0, "should be same as owner is not considered");
```

Comment:
In the documentation for the function it says 

> If the name is equal then:
>     No owner will be found to be smaller than every other owner. If both don't have an owner, 0 is returned.

but also

>     the owner will only be used if the names are equal.

Apparently the way `keyCmp` currently works, is by comparing the owner as well. (I actually should add a test case for different owners with different keys now that I think about it...)

Am I even correct in the assumption that in this Key name: `user:user_a/a` `user_a` would be the owner? Is this specific to user keys only?

Reply:
```
	keySetName (k1, "user:user_a/b");
	keySetName (k2, "user:user_b/a");
	succeed_if (keyCmp (k1, k2) > 0, "should be same as owner is not considered");
	succeed_if (keyCmp (k2, k1) < 0, "should be same as owner is not considered")
```

This would also fail

If I am correct about owners, shouldn't this also succeed because b > a?

Reply:
The concept of "owners" were removed, so these distinctions are not relevant anymore.